docs/cli-quickstart.md

```diff
@@ -41,7 +42,13 @@ In your terminal, navigate to the directory where you'd like to create your proj
 mkdir hello_sphinx && cd hello_sphinx
 ```
 
-## 3. Install dependencies
+## 3. Update Foundry
+Make sure you have the latest version of Foundry installed:
+```
+foundryup
+```
```

Comment:
It was kinda nice when we had the `--foundryup` flag in the `sphinx init` command to save a step, but I'm not opinionated. Your choice